spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -52,6 +52,8 @@
   it_behaves_like 'redundant', '(true)', 'true', 'a literal'
   it_behaves_like 'redundant', '(false)', 'false', 'a literal'
   it_behaves_like 'redundant', '(/regexp/)', '/regexp/', 'a literal'
+  it_behaves_like 'plausible', '("x"; "y")'
```

Comment:
Are those examples good? The parentheses look redundant to me here, as those could be two independent expressions that happen to be wrapped in parens...